docs/external-node.md

```diff
@@ -0,0 +1,577 @@
+is attached on OVS as uplink, and a new OVS internal Port is added to take over
+the uplink interface's IP/MAC and routing configurations. On Windows, the DNS
+configurations are also moved to the OVS internal port from uplink. Before
+attaching the uplink to OVS, the network interface is renamed with a suffix
+"~", and OVS internal port is configured with the original name of the uplink.
+As a result, IP/MAC/routing entries are seen on a network interface configuring
+with the same name on the external Node.
+
+The outbound traffic sent from the external Node enters OVS from the internal
+port, and finally output from the uplink, and the inbound traffic enters OVS
+from the uplink and output to the internal port. The IP packet is processed by
+the OpenFlow pipeline, and the non-IP packet is forwarded directly.
+
+Following is a diagram for the traffic model on an external Node.
+![Traffic On ExternalNode](assets/traffic_external_node.svg)
+
+## RBAC for antrea-agent
+
+An external Node is thought as an untrusted point on the network. To follow the
+security principle that least permissions are given, the RBAC for `antrea-agent`
+running on `ExternalNode` feature is modified:
+
+- Only `get`, `list` and `watch` permissions are given on resource `ExternalNode`
+- Only `update` permission is given on resource `antreaagentinfos`, and `create`
+  permission is moved to `antrea-controller`
+- for more details please refer to [vm-agent-rbac.yaml](/build/yamls/externalnode/vm-agent-rbac.yml)
+
+`antrea-agent` reports its status by updating the `antreaagentinfo` resource
+which is configured with the same name as the `ExternalNode`. `antrea-controller`
+creates an `antreaagentinfo` object for each new `ExternalNode`, and then
+`antrea-agent` updates it every minute with its latest status. `antreaagentinfo`
+is deleted by `antrea-controller` when the `ExternalNode` is removed.
+
+## Apply Antrea NetworkPolicy to ExternalNode
+
+### Antrea NetworkPolicy configuration
+
+An Antrea NetworkPolicy is applied to an `ExternalNode` by providing an
+`externalEntitySelector` in the `appliedTo` field. **The `ExternalNode` object
+is automatically converted to `ExternalEntities`**, which are used by
+`antrea-controller` to calculate the NetworkPolicy span, and `antrea-agent`
+running on the `ExternalNode` receives the internal Antrea NetworkPolicy
+objects.
+
+Following types of network peers are supported when applying an Antrea
+NetworkPolicy to an external Node:
+
+- ExternalNode with an `externalEntitySelector`
+- CIDR with an `ipBlock`
+- fqdn address in an egress rule
+
+Following actions are supported when applying an Antrea NetworkPolicy to an
+external Node:
+
+- Allow
+- Drop
+- Reject
+
+Following is an example of applying an Antrea NetworkPolicy to the `ExternalNode`
+objects labeled with `role=db` to reject SSH connections from IP "172.16.100.5"
+or from other external Nodes labeled with `role=front`:
```

Comment:
Use the same term here and in line 451; either "external Nodes" or "ExternalNodes".